src/screenshotselectareagraphicsview.cpp

```diff
@@ -42,8 +42,8 @@ void ScreenshotSelectAreaGraphicsView::mousePressEvent(QMouseEvent *event)
       color.setAlpha(128);
       QBrush brush(color);
       selectedAreaRect_ = scene()->addRect(QRectF(), pen, brush);
-    } 
-    selectedAreaRect_->setRect(QRectF(p0_,QPointF(event->pos())).normalized());
+    }
+    selectedAreaRect_->setRect(QRectF(p0_,event->position()).normalized());
```

Comment:
The painted preview rect should also have margins. You can see the difference in position by comparing the drawn edges at the far bottom-right with the top-left.

Reply:
If by "should" you mean it has a *visible* margin, that may be a result of rounding because It's a `QRectF`. But if you mean we should add a margin here, no, we shouldn't, again because it's a `QRectF`.

Anyway, if there's also another miscalculation, it should be found and fixed in another PR.

Reply:
Also, see https://doc.qt.io/qt-6/qgraphicsrectitem.html#details. That's exactly what the code does. Qt is responsible for a probable inconsistency. Since we don't know when it might be corrected, we can't add margins. Fortunately, it isn't important for us in practice either.

This PR was about *our* miscalculation.